h3m/h3mlib/h3m_structures/object_details/h3m_od_body_ext_quest.h

```diff
@@ -68,6 +57,27 @@ struct H3M_OD_BODY_EXT_QUEST {
     struct H3M_OD_BODY_EXT_QUEST_DEADLINE_AND_MESG deadline_and_mesg;
 };
 
+union H3M_OD_BODY_EXT_REWARD {
```

Comment:
Wrong indentation but good addition. Looks correct but needs testing to make sure it is correct for all types